eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/rest/ReservationResource.java

```diff
@@ -17,35 +18,38 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-@RequestMapping("/api")
+@RequestMapping("/api/reservations")
 @AllArgsConstructor
 public class ReservationResource {
 
   private ReservationService reservationService;
 
-  @GetMapping("/reservations")
-  public List<Reservation> findAllReservations() {
-    return reservationService.findAllReservations();
+  @GetMapping
+  public ResponseEntity<List<Reservation>> findAll() {
+    List<Reservation> reservations = reservationService.findAll();
+    return new ResponseEntity<>(reservations, HttpStatus.OK);
   }
 
-  @GetMapping("/reservations/{id}")
-  public Reservation findOneReservationById(@PathVariable Long id) {
-    return reservationService.findOneReservationById(id);
+  @GetMapping("{id}")
+  public ResponseEntity<Reservation> findOne(@PathVariable Long id) {
+    Optional<Reservation> reservationOptional = reservationService.findOne(id);
+    return reservationOptional.isEmpty() ? ResponseEntity.notFound().build()
+        : ResponseEntity.ok(reservationOptional.get());
   }
 
-  @PostMapping("/reservations")
-  public ResponseEntity<?> createReservation(@RequestBody Reservation reservation) {
+  @PostMapping
+  public ResponseEntity<Reservation> createReservation(@RequestBody Reservation reservation) {
     reservationService.createReservation(reservation);
```

Comment:
Ez mehet a ResponseEntity-be

Reply:
Elvileg nekem így van és minden up to date meg a mstert is mergeltem, de itt nem írja, hogy outdated :)
```java
@PostMapping
  public ResponseEntity<Reservation> createReservation(@RequestBody Reservation reservation) {
    reservationService.createReservation(reservation);
    return new ResponseEntity<>(reservation, HttpStatus.CREATED);
  }

Reply:
Ja lehet, hogy csak a problémát nem értettem :)